### PR TITLE
Add episode 2 - testing

### PR DESCRIPTION
**Pull request by smillie**

### CODE REVIEW

**Comment by smillie** [comment]
If we decide that we need a slight edit around 41:49 the outro timestamp will need changed.

File: content/episode/002-testing.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: "Testing"
+episode: "2"
+Description: "Al and Andrew talk about testing. What is it, why is it useful, who should do it, etc..."
+podcast: "pdd-2.mp3"
+podcast_bytes: "27717141"
```
Comment (McInkay) [inline content/episode/002-testing.md]:
```suggestion
podcast_bytes: "27684986"
```

File: content/episode/002-testing.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+podcast_duration: "00:43:58"
```
Comment (McInkay) [inline content/episode/002-testing.md]: Weirdly, this is the same length now :-\ but I don't care, it's just silence at the end. I'll figure that out for the next episode.

File: content/episode/002-testing.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Timings
+
+[00:28] Introduction to testing
+[06:10] Types of Testing
+[13:30] The role of testers
+[19:50] Test case definition
+[26:01] Test driven development
+[36:57] Code coverage measurements
+[42:55] Outro
```
Comment (McInkay) [inline content/episode/002-testing.md]:
```suggestion
[42:28] Outro
```

**Comment by smillie** [comment]
accepted suggested edits

**Pull request merged**